components/game/game.js

```diff
@@ -118,14 +151,31 @@ const Game = () => {
       {deckIsShuffled &&
         <div>
           <h2 className={gameStyles.turnIndicator}>{playerTurn === 0 ? 'Your' : `Player ${playerTurn + 1}'s`} turn.</h2>
+          <h2>{`Select a combo thats fits ${selectCombo}`}</h2>
```

Comment:
the whole thing doesn't need to be in string literals. in fact, i think only the variable is needed, no string literals at all

```suggestion
          <h2>Select a combo thats fits {selectCombo}</h2>
```